packages/auto-form/yalc.lock

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file isn't needed. PR got merged, can you open a new one to remove it please?

Reply:
Whoops sorry, I can open a PR to remove it if you'd like @seanforyou23 ?

Reply:
https://github.com/syndesisio/syndesis-react-poc/pull/105